examples/remoting/stateful_remote_spawn.cpp

```diff
@@ -0,0 +1,181 @@
+// This program illustrates how to spawn a simple aggregator
+// across the network.
+//
+// Run server at port 4242:
+// - remote_spawn -s -p 4242
+//
+// Run client at the same host:
+// - remote_spawn -H localhost -p 4242
+
+#include "caf/io/middleman.hpp"
+
+#include "caf/actor_from_state.hpp"
+#include "caf/actor_system.hpp"
+#include "caf/actor_system_config.hpp"
+#include "caf/caf_main.hpp"
+#include "caf/scoped_actor.hpp"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+struct aggregator_trait {
+  using signatures = caf::type_list<caf::result<void>(caf::add_atom, int32_t),
+                                    caf::result<int32_t>(caf::get_atom)>;
+};
+
+using aggregator = caf::typed_actor<aggregator_trait>;
```

Comment:
```suggestion
using aggregator_actor = caf::typed_actor<aggregator_trait>;
```

We usually follow this naming pattern:

- state class: `<thing>_state`
- trait class: `<thing>_trait`
- typed handle: `<thing>_actor`
- instance: `<thing>`

So in this case:
- `struct aggregator_trait ...`
- `using aggretator_actor = ...`
- `auto aggregator = sys.spawn(actor_from_state<aggretator_state>)`